tests/api-tests/queries/search.test.js

```diff
@@ -134,5 +134,38 @@ multiAdapterRunners().map(({ runner, adapterName }) =>
         expect(data.allNumbers).toEqual([]); // No results
       })
     );
+
+    test.only(
+      'empty string',
+      runner(setupKeystone, async ({ keystone }) => {
+        const create = async (listKey, item) => createItem({ keystone, listKey, item });
+        await Promise.all([
+          create('Test', { name: 'one' }),
+          create('Test', { name: '%islikelike%' }),
+          create('Test', { name: 'three' }),
+          create('Number', { name: 12345 }),
```

Comment:
I don't understand why this is here? You're only querying things on the Test list?

Reply:
Mostly for consistency. If I could be bothered I'd hoist this into a common setup function that was used by all of these tests.